sample-apps/blank-java/pom.xml

```diff
@@ -23,24 +23,9 @@
       <version>2.2.9</version>
     </dependency>
     <dependency>
-      <groupId>com.amazonaws</groupId>
-      <artifactId>aws-lambda-java-log4j2</artifactId>
-      <version>1.5.0</version>
-    </dependency>
-    <dependency>
-      <groupId>org.apache.logging.log4j</groupId>
-      <artifactId>log4j-api</artifactId>
-      <version>[2.17.1,)</version>
-    </dependency>
-    <dependency>
-      <groupId>org.apache.logging.log4j</groupId>
-      <artifactId>log4j-core</artifactId>
-      <version>[2.17.1,)</version>
-    </dependency>
-    <dependency>
-      <groupId>org.apache.logging.log4j</groupId>
-      <artifactId>log4j-slf4j18-impl</artifactId>
-      <version>[2.17.1,)</version>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-nop</artifactId>
```

Comment:
This dependency silently discards log messages. I don't think it's a good idea to use it for an example.

Reply:
I actually want to remove SLF4J from the dependencies and just use the context logger (SLF4J is required for the testing to work, so I include it for test context only). However, I can't do so because I think another AWS dependency that is pulling in SLF4J somewhere, causing this warning in logs (https://stackoverflow.com/questions/7421612/slf4j-failed-to-load-class-org-slf4j-impl-staticloggerbinder). The warning is resolved with this dependency. That being said, it's just a warning but I wanted all the output and logs to look as clean as possible.

Reply:
I think it would be better to include a real implementation of slf4j backend. sl4j-simple for example.